auth/index.js

```diff
@@ -8,7 +8,10 @@ const { JWT_CERT } = process.env;
 
 export function verifyWebToken(ctx) {
   log('Verifying token');
-  const token = ctx && ctx.headers && ctx.headers.authorization;
+  const token =
+    ctx &&
+    ctx.headers &&
+    (ctx.headers.Authorization || ctx.headers.authorization);
```

Comment:
But what do we do if both "Authorization" and "authorization" are present? I'm not sure, the spec says to lowercase the headers field names, which we could do but we'd still need to decide what to do with duplicates. Open to suggestions.

Reply:
For duplicates, I think its fair to stick to the lower-case values. And as of now, we being the only-consumers need to take care of this client side too.

Reply:
In that case I would pass all the headers though a util and turn them all to lowercase.

Reply:
I'd rather not mess with the headers manually, and have us support both mix and all lower. I've changed it to prefer all lower, which would be what most clients would set and many libraries normalise to anyway.